instance.tf

```diff
@@ -45,6 +46,15 @@ resource "aws_security_group" "web" {
     cidr_blocks = ["108.255.250.119/32"]
   }
 
+  ingress {
+    description = "HTTPS ingress"
+    from_port   = 443
```

Comment:
change port to 80 as we are not yet using certificate manager.
Application can only be access via port 80 for now